src/astro/files/type/__init__.py

```diff
@@ -44,7 +44,16 @@ def get_filetype(filepath: Union[str, pathlib.PosixPath]) -> FileTypeConstants:
     if isinstance(filepath, pathlib.PosixPath):
```

Comment:
Should `type` be renamed to `types` in module name like `files` and `databases`? That can happen in separate PR too

Reply:
Ya, I was thinking the same. It collides with python's type() BIF.